src/main/java/com/car/app/boundary/CarResource.java

```diff
@@ -32,6 +40,14 @@
 	@Inject
 	private ICarService carService;
 	
+	@Resource(mappedName = "jms/queue")
```

Comment:
Mixing in the API layer details of the queue is not very "clean", although it works, it would be "cleaner" if this queue related code was into a dedicated class or maybe in the service layer.